src/Reports/PagesWithoutReviewScheduleReport.php

```diff
@@ -125,16 +125,18 @@ public function sourceRecords($params = [])
             ));
         }
 
-        $records->sort("ParentID");
-        $records = $records->toArray();
+        if ($sort !== null) {
```

Comment:
```suggestion
        if ($sort === null) {
```

Reply:
Note that this suggestion was for when the check looked like this:
```php
if ($sort !== null) {
    $sort = 'ParentID';
}
```

That is no longer what the check is for, so this suggestion no longer applies.